slack-api-model/src/test/java/test_locally/api/model/event/LinkSharedEventTest.java

```diff
@@ -52,7 +52,7 @@ public void serialize() {
         Gson gson = GsonFactory.createSnakeCase();
         LinkSharedEvent event = new LinkSharedEvent();
         String generatedJson = gson.toJson(event);
-        String expectedJson = "{\"type\":\"link_shared\"}";
+        String expectedJson = "{\"type\":\"link_shared\",\"is_bot_user_member\":false}";
```

Comment:
The generated JSON string here consists of non-null values. As `is_bot_user_member` always exists, I chose the primitive value. That's why this test needs to be updated this way.